Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -131,6 +131,16 @@ script:
       name: event_limit
       required: false
       secret: false
+    - name: batch_limit
```

Comment:
not sure that i get the difference between batch_limit and event limit

Reply:
It's helping performance in some way, let's discuss